### PR TITLE
support parsing multiple packages per line

### PR DESCRIPTION
**Pull request by vogler**
.merlin allows multiple packages per line, so
~~~
PKG foo bar
~~~
instead of
~~~
PKG foo
PKG bar
~~~
should also be supported.

### CODE REVIEW

File: src/merlin_parser.ml [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
+let pkg_line = string "PKG " *> sep_by (char ' ') (take_while1 pkg_name)
+let skip_line = skip_while is_not_end_line *> return []
 
 let any_line = pkg_line <|> skip_line
 
-let parser = sep_by linebreak any_line
+let parser = sep_by (char '\n') any_line
 
 let parse content =
+  print_endline "hallo";
```
Comment (Leonidas-from-XIV) [inline src/merlin_parser.ml]: Line left over.

**Review by Leonidas-from-XIV — CHANGES_REQUESTED** [review]
Thanks, that's a very useful addition. I found some useless code, after that I'm ok with merging it.

File: src/merlin_parser.ml (hunk 1 above)
Comment (vogler) [reply src/merlin_parser.ml]: Ups, sorry 😄

**Review by Leonidas-from-XIV — APPROVED** [review]

**Comment by Leonidas-from-XIV** [comment]
Thanks! I'll release it after the angstrom 0.7 release hits OPAM.

**Pull request merged**